src/DevChatter.DevStreams.Web/Pages/Privacy.cshtml

```diff
@@ -4,5 +4,28 @@
     ViewData["Title"] = "Privacy Policy";
 }
 <h2>@ViewData["Title"]</h2>
+<br />
 
-<p>Use this page to detail your site's privacy policy.</p>
\ No newline at end of file
+<p>
+    Here on Devchatter we are committed to safeguarding, with our Dino<img src="https://cdn.discordapp.com/emojis/422200063180406812.png?v=1" style="max-height: 28px; max-width: 28px;" alt="DinoHypeSquad" />
```

Comment:
Should we put this image on the site instead of referencing the discord cdn, since we don't control that.